src/middlewares/task.middleware.js

```diff
@@ -158,6 +150,20 @@ export default (store: any) => (next: any) => (action: any) => {
       next(attachTaskMetadata(task, child.pid));
 
       child.stdout.on('data', data => {
+        // The 'eject' task prompts the user, to ask if they're sure.
+        // We can bypass this prompt, as our UI already has an alert that
+        // confirms this action.
+        // TODO: Eject deserves its own Redux action, to avoid cluttering up
+        // this generic "RUN_TASK" action.
+        // TODO: Is there a way to "future-proof" this, in case the CRA
```

Comment:
I checked [`react-scripts`'s eject code](https://github.com/facebook/create-react-app/blob/92d9c5ada40c815dde8948acb96ccbd5636a4e7f/packages/react-scripts/scripts/eject.js#L41) and seems like the prompted question is hard-coded and there is not any parameters  for passing prompts (something like `--force` or `--yestoall`). I don't think it is something to change by its own. If there is going to be a change I think it would effect multiple places so I don't think we should worry about it for now.